### PR TITLE
add angular-timer

### PR DESCRIPTION
**Pull request by lookfirst**

### CODE REVIEW

File: package-overrides/github/siddii/angular-timer@1.3.3.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{  
+  "main":"dist/angular-timer",
+  "registry":"jspm",
+  "format":"global",
+  "dependencies":{  
+    "angular":"*",
+    "moment":"*",
+    "humanize-duration":"*"
```
Comment (guybedford) [inline package-overrides/github/siddii/angular-timer@1.3.3.json]: Do you want to version range moment and humanize-duration?

**Comment by guybedford** [comment]
Amazing to see you've mastered these :)

File: package-overrides/github/siddii/angular-timer@1.3.3.json (hunk 1 above)
Comment (lookfirst) [inline package-overrides/github/siddii/angular-timer@1.3.3.json]: Nah... that's what CI is for. ;-)

Comment (guybedford) [inline package-overrides/github/siddii/angular-timer@1.3.3.json]: At least lock down the major - `"moment": "2"` etc?

**Comment by lookfirst** [comment]
lol... i'm still not very happy about it, but no time / energy to really fix it... =)

**Comment by guybedford** [comment]
It would be a big help, nice to know that things will work indefinitely.

**Comment by lookfirst** [comment]
ping!

**Comment by lookfirst** [comment]
@guybedford ping

**Comment by guybedford** [comment]
Sorry missed that you'd added them already!

**Pull request merged**